Systems/c172p-engine.xml

```diff
@@ -79,6 +79,60 @@
             </clipto>
             <output>/engines/active-engine/rpm-over-redline</output>
         </fcs_function>
+
+        <fcs_function name="damage level calculation">
+              <function>
+                <ifthen>
+                    <eq>
```

Comment:
Is the `<eq>` needed? Isn't just `<p>` enough (it should be if it's a boolean). Maybe rename it to `damage-allowed`.

Reply:
Richard helped me with this, he used this in his example. As for the name, alright, will do.

Reply:
@onox regarding the name, all the other allowed properties use an underscore. 
See engine.xml lines 696 to 717

Reply:
I checked without <eq>, did not work.